### ISSUE TITLE
Mathfonts of \tau

### ISSUE THREAD

**Issue by JuntongLai** [HaivuUK/LaTeX-Guides#1]
Hi [HaivuUK](https://github.com/HaivuUK),

Thank you so much for bringing this excellent LaTeX guide.

I am just checking the guide because I found the math font of $\tau$ is a bit different from the one by default rendered by pdflatex.

By default, it is ⲧ. To change the style of `\tau`, what I could do was to use LuaLaTeX to do math symbols with the font STIX:
```latex
\usepackage{unicode-math}
\setmathfont{STIX Two Math}
```
Would you have any suggestions?

PS: I have been thinking it may be a better way to ask you questions on LaTeX so we can record those here and reflect them to the guide if helpful.

 https://github.com/HaivuUK/LaTeX-Guides/blame/d343f3cc1da2107bf8539498819443085cd85108/docs/maths.md#L32

**Comment by HaivuUK** [comment]
Hi @LaiDream 

Thank you for raising this as an issue. Having a public record is good.

Yes, the font will be different to the LaTeX default (_computer modern_).

The site does not directly use LaTeX for rendering but instead uses the MathJax JavaScript library which has its own default font. However, to my understanding, the default is supposed to be close to _computer modern_. Currently, there is a limitation to MathJax3 in that it does not provide additional fonts, and this will not be resolved till the MathJax4 release.

The site rendering:
![image](https://github.com/user-attachments/assets/44ed328a-b84b-42df-820c-7881140586f7)

pdfLaTeX using computer modern font:
![image](https://github.com/user-attachments/assets/20572fce-f9e1-4e61-9be4-244eac8d7b1a)

pdfLaTeX using stix2 font:
![image](https://github.com/user-attachments/assets/3abe4c4c-dfc3-4343-b924-e95e30067816)

@LaiDream do you see something different on your web browser? If so, could you post images?

If you wanted to use the stix font with pdflatex you can do the following:
```
\documentclass{article}

\usepackage[notext]{stix2} % notext option prevents stix from changing your main text font

\begin{document}
	This a test of some text to then put some maths in line \(\tau\) such as that.
	
	And then some display maths:
	\[
	\tau = 2\pi
	\]
\end{document}
```

Even though there is a font section in the guide, I realise I have not covered mathfonts, and it is a good idea to include. This should help clear up possible confusion, and make it clear how to control it in pdflatex and other compilers. A full list of available fonts can be found here, [CTAN Math Fonts](https://ctan.org/topic/font-maths), and it is really a case of picking your favourite with the coverage you require.

**Comment by JuntongLai** [comment]
Hi [HaivuUK](https://github.com/HaivuUK),

Thank you!!

They look like these below from my site:
![image](https://github.com/user-attachments/assets/f0500dae-9b52-432e-a208-0250cf6d4eaf)

Do they look different from yours?

**Comment by HaivuUK** [comment]
This issue was related to the different results produced by microsoft office applications when rendering rather than LaTeX directly and will be marked as closed for now.

The issue being an equation rendered within microsoft powerpoint using _Cambria Math_ did not match the standard maths font in LaTeX. The solution is to use [stix2-otf](https://ctan.org/pkg/stix2-otf/?lang=en) or [stix2-t1](https://ctan.org/pkg/stix2-type1?lang=en) to match _Cambria Math_.

**Comment by HaivuUK** [comment]
Specifically addressed in commit c90fbee1564ca614925e42f65dd05269836af278 
https://github.com/HaivuUK/LaTeX-Guides/blame/c70a09365277acf3f45384fc2d71a36b12261355/docs/fonts.md#L209
